test/IIF4DoorsTests.jl

```diff
@@ -0,0 +1,81 @@
+using GraphPlot
+using Neo4j
+using DistributedFactorGraphs
+using IncrementalInference
+
+@testset "test fourdoor early example" begin
```

Comment:
Left this as it can be used as template for end to end tests.

Reply:
separate discussion on how cyclic IIF dependency can be included for tests -- decision thus far was to add conditional tests.